hydroshare/local_settings.template

```diff
@@ -184,6 +184,12 @@ HSWS_TIMEOUT = 3
 HSWS_PUBLISH_URLS = False
```

Comment:
Please work with Dan / Martin and ensure they understand and update any necessary deployment scripts or manual steps